### ISSUE TITLE
[BUG] H011 should only be matching attributes that are inside a valid html tag

### ISSUE THREAD

**Issue by christopherpickering** [Riverside-Healthcare/djLint#42]
H011 should not be matching the id attributes here.
```html
        {% filter heading(heading_level,
            role="data" if obj == module else "attr",
            id=html_id,
            toc_label=attribute.path,
            hidden=True) %}
        {% endfilter %}
```

**Comment by christopherpickering** [comment]
Related to #37

**Comment by pawamoy** [comment]
I had also opened #38 for this :)

**Comment by christopherpickering** [comment]
closed as dup of #38
